maps/src/pointing.cxx

```diff
@@ -280,7 +280,7 @@ get_fk5_j2000_to_gal_quat()
 
 static void
 create_det_az_el_trans(const G3Timestream &az, const G3Timestream &el,
-    G3VectorQuat &trans_quats)
+    G3VectorQuat &trans_quats) // XXX: switch to G3TimestreamQuat?
```

Comment:
Why not?

Reply:
Changing all of these causes API churn in the old mapmaker, which works in terms of G3VectorQuat. I'd kind of like to keep this as-is unless we can rm the old mapmaker, then update it.

Reply:
Fair enough.  I'd like to nuke this whole file altogether in the long run...